src/daemon/daemon.cpp

```diff
@@ -466,7 +466,7 @@ void validate_image(const mp::LaunchRequest* request, const mp::VMImageVault& va
     catch (const mp::IncompatibleWorkflowException&)
```

Comment:
Shall we change the exception name to `mp::IncompatibleBlueprintException`?

Reply:
There is a bunch of renaming to do with the implementation and that is tracked in #2497.  Since this particular PR is *only* for the release and only affects user facing strings, we should wait on this.